ala-name-matching-search/src/test/java/au/org/ala/names/search/AutocompleteTest.java

```diff
@@ -150,13 +151,13 @@ public void testAutocomplete12() throws Exception {
         List<Map> results = searcher.autocomplete("rush li", 10, true);
         assertNotNull(results);
         assertTrue(results.size() > 2);
-        Map syn = results.get(2);
-        assertEquals("Sisyrinchium rosulatum", syn.get("name"));
-        List<Map> synonyms = (List<Map>) syn.get("synonymMatch");
-        assertNotNull(synonyms);
-        assertTrue(synonyms.size() > 0);
-        Map synonym = synonyms.get(0);
-        assertEquals("Yellow Rush Lily", synonym.get("commonname"));
+        Optional<Map> syn = results.stream().filter(r -> "Sisyrinchium micranthum".equals(r.get("name"))).findFirst();
+        assertTrue(syn.isPresent());
+        //List<Map> synonyms = (List<Map>) syn.get().get("synonymMatch");
+        //assertNotNull(synonyms);
+        //assertTrue(synonyms.size() > 0);
+        //Map synonym = synonyms.get(0);
```

Comment:
Dave prefers deleting code as git history can be used to get it back

Reply:
Removed.